README.md

```diff
@@ -1,10 +1,31 @@
 # Arachnivania
 
-This project is a Godot-powered game using the native GDscript language. Art is created using Aseprite.
+This project is a Godot-powered game using the native GDscript language. Art is created using Aseprite, TileSetter, and GIMP.
+
+[1. Description](#desription)
+[2. Controls](#controls)
+[3. Build Info](#build-info)
+[4. Authors](#license)
+[5. License](#license)
+[6. Acknowledgements](#acknowledgements)
 
```

Comment:
Do we need Authors, License and Ack. Is it possible to link to the actual file?

Reply:
Oh yeah, you can link to whatever files you want with Markdown. What those links do at the top is scroll to the section of the page that is linked. So if you click on Acknowledgements at the top, its scrolls to the Acknowledgements part of the README

Reply:
Oops wrong comment spot

Reply:
Yeah link them to the actual files instead of putting that info in the ReadMe